docs/policies/expense_policy.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Expense and Reimbursement Policy'
+---
+# AlmaLinux Expense and Reimbursement Policy
+
+This policy was adopted by the Board of Directors at [Board meeting #39 in April of 2025](/Transparency.html#minutes-of-almalinux-os-foundation-board-meetings). 
+
+### 1. Purpose
+
+This policy establishes the procedure for reimbursing travel, accommodation, and other related expenses incurred by members, contributors, or employees of the AlmaLinux OS Foundation (Foundation) during conferences, events, or activities that align with the Foundation's mission.
+
+### 2. Scope
+This policy applies to:
+
+-   Foundation staff
+-   Official contributors and volunteers
+-   Board members
+-   Other individuals who are authorized to travel on behalf of the Foundation
```

Comment:
Just being picky, realised that some bullet points end up with dot and some don't